themes/root/templates/Helpers/icons/svg-sprite.phtml

```diff
@@ -0,0 +1,3 @@
+<svg class="icon--svg"<?=$this->attrs ?> aria-hidden="true">
+    <use xlink:href="<?=$this->imageLink($src) ?>#<?=$this->icon ?>"></use>
```

Comment:
Another case where I wonder if we should `escapeHtmlAttr` the `$this->icon` value out of caution.